lib/ansible/modules/cloud/openstack/os_port.py

```diff
@@ -291,6 +302,7 @@ def main():
         device_owner=dict(default=None),
         device_id=dict(default=None),
         state=dict(default='present', choices=['absent', 'present']),
+        port_security_enabled=dict(default=None, type='bool')
```

Comment:
I guess the default for this should be True, shouldn't it?

Reply:
It is true.
When not providing any value regarding this argument, OpenStack automatically set it as True.

Reply:
Hello @jamescassell ,

Is it acceptable for you that the argument default is None?
I think if the argument is not used, we should rely on the neutron default behavior.

Reply:
I'm fine with shipping as is, but i'm not the best person to say.

Reply:
This can be set on OpenStack network level also. So my thought would be that the default should be None, in which case the port will inherit the setting from the network. This is similar to how OpenStackSDK itself, and "openstack port-create" CLI, behave. I guess that setting the default to True would override that always, and enable it even if it the user did not specify it and the network default was set to "False". Also, in case of port update we presumably want to leave this parameter as-is unless it is explicitly specified.